WordPress/src/main/java/org/wordpress/android/ui/posts/PostListMainViewModel.kt

```diff
@@ -23,10 +47,24 @@ class PostListMainViewModel @Inject constructor() : ViewModel() {
     private val _isFabVisible = MutableLiveData<Boolean>()
     val isFabVisible: LiveData<Boolean> = _isFabVisible
 
+    private val _selectTab = SingleLiveEvent<Int>()
+    val selectTab = _selectTab as LiveData<Int>
+
+    private val _scrollToLocalPostId = SingleLiveEvent<Int>()
+    val scrollToLocalPostId = _scrollToLocalPostId as LiveData<Int>
+
+    private val _snackBarMessage = SingleLiveEvent<UiString>()
```

Comment:
Should we use [SnackbarMessageHolder](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/pages/SnackbarMessageHolder.kt) instead? We can update it to use `UiString`, possibly in a different PR against `develop` and then bring that change into this branch. What do you think?

Reply:
👍